dask/array/tests/test_cupy.py

```diff
@@ -214,7 +214,7 @@ def test_basic(func):
     ddc = func(dc)
     ddn = func(dn)
 
-    assert type(ddc._meta) is cupy.core.core.ndarray
+    assert type(ddc._meta) is cupy.ndarray
```

Comment:
Just for my own understanding, is this backwards compatible for `cupy < 9`? I suspect that it is, but wanted to double check

Reply:
Yep it is. That's probably what we should have done before since `core` is viewed as internal. In any event it is now fixed :)